### DISCUSSION TITLE
Issue bundling dependency that exports "as default"

### DISCUSSION THREAD

**Discussion by bvkimball** [egoist/tsup#873] [Q&A]
I am using a dependency that export its default with that "as" syntax.

```
export { borderBoxWarn, useDimensions as default, observerErr };
```

At runtime i get an error: `TypeError: (0 , Ui.default) is not a function`. 

`Ui` which is the imported library is equal to the object `{ borderBoxWarn: xxx, default: { default: () => xxx , ...}, observerErr:xxx };`

not sure if there is a way to get that to work.

**Comment by stefvw93** [comment]
Did you ever resolve this?
